README.md

```diff
@@ -8,6 +8,6 @@ Library for validating certificates.
 <dependency>
     <groupId>no.digipost</groupId>
     <artifactId>certificate-validator</artifactId>
-    <version>1.0.4</version>
+    <version>1.0.5</version>
```

Comment:
May consider just linking to https://search.maven.org/artifact/no.digipost/certificate-validator to avoid having to remember to update this.

Like this: https://github.com/digipost/signature-api-client-java/commit/e8d9b191f13c8e66e35492591d6934cde16c8d0e

Reply:
Seems better! I'll fix :)